### DISCUSSION TITLE
Demakes

### DISCUSSION THREAD

**Discussion by RobLoach** [nesbox/TIC-80#1287] [Show and tell]
What are some of your favourite demakes/remakes in TIC-80? By demake/remake, I mean a recreation of an existing game ported to TIC-80, keeping similar gameplay, level design, etc.

- [Super Meat Boy](https://tic80.com/play?cart=1512)
- [Mario Bros](https://tic80.com/play?cart=223)
- [VVVVVV](https://tic80.com/play?cart=627)
- [Dangerous Dave](https://tic80.com/play?cart=880)

What's a game you'd like to see de-made or ported to TIC-80?

**Comment by 12buntu** [comment]
It may not be possible, but I would really love to see something like Starfox(/Starwing) made on TIC-80. I'm not attached to that game in particular, but I love "3D" games from that era.

**Comment by desC-37** [comment]
god of war, not complete, obviously. :D

**Comment by darltrash** [comment]
Mari0 would be interesting to see

**Comment by atesin** [comment]
i am trying to make a near faithful demake of TIME PILOT (i am not a programmer)... anyway, some of the **arcade** demakes i would like to see and play are (oh so many! :D) :
- time pilot
- bosconian
- rygar
- rastan
- double dragon .. or other isometric sidescroller like
  - captain commando
  - cadillacs and dinosaurs
  - knights of the round
  - renegade / kunio-kun
  - golden axe
- r-type
- montezuma (atari xl/xe)
- forth apocalypse (atari xl/xe)
- bad dudes vs dragon ninja
- raiden / twin cobra
- xain'd sleena
- athena
- pole position
- out run
- gradius
- rally-x
- darwin 4078
- valtric
- section-z
- elevator action
- etc... i love videogames
